pytest_html/plugin.py

```diff
@@ -41,6 +41,9 @@ def pytest_addoption(parser):
     group.addoption('--html', action='store', dest='htmlpath',
                     metavar='path', default=None,
                     help='create html report file at given path.')
+    group.addoption('--self-contained-html', action="store", dest='self_contained_html', default=False,
```

Comment:
@davehunt okay, I think this adds an option, but how can I access this option latter in the code